std_msgs/Header header
detector_ros/ShapeDetection[] detections